platform/ios/src/MGLMapView.mm

```diff
@@ -2478,7 +2478,7 @@ - (void)showAttribution:(id)sender
     }];
     [attributionController addAction:telemetryAction];
     
-    NSString *cancelTitle = NSLocalizedStringWithDefaultValue(@"CANCEL", nil, nil, @"Cancel", @"");
+    NSString *cancelTitle = NSLocalizedStringWithDefaultValue(@"CANCEL", nil, nil, @"Cancel", @"Cancel");
```

Comment:
```suggestion
    NSString *cancelTitle = NSLocalizedStringWithDefaultValue(@"CANCEL", nil, nil, @"Cancel", @"Title of button for dismissing attribution action sheet");
```

Reply:
Remember to synchronize Localizable.strings with this change:

https://github.com/mapbox/mapbox-gl-native/blob/9bd59bf041b0d0ca400f586fdbfed53d06258613/platform/ios/resources/Base.lproj/Localizable.strings#L4